docs/HOWTO_AZURE_DEPLOY.md

```diff
@@ -0,0 +1,149 @@
+- Start at Step 1 for installing Node.js
+- Install Node: `sudo apt install -y nodejs`
+- DON’T DO THIS LINE: Install Yarn: sudo apt-get update && sudo apt-get install yarn
+- INSTEAD USE THE NPM PACKAGE MANAGER `sudo npm install -g yarn`
+- IF YOU ACCIDENTALLY USED THE LINE WE TOLD YOU NOT TO, HERE’S HOW TO REMOVE IT: `sudo apt remove yarn cmdtest`
+- Clone down Spoke `git clone https://github.com/MoveOnOrg/Spoke.git`
+
+## PACKAGE
+### Spoke installation and package setup
+- CD into Spoke and 
+- Run `yarn install`
+- Begin the [Spoke Minimalist Deploy](https://github.com/MoveOnOrg/Spoke/blob/main/docs/HOWTO_MINIMALIST_DEPLOY.md) 
+
+### If you run `yarn run prod-build-client` GAVE AN ERROR OF FAILING TO LOAD ‘/home/{username}/Spoke/webpack/config.js’
+- Install Docker for preliminary database and cache setup, [linked here](https://docs.docker.com/engine/install/ubuntu/)
+- Verify your docker is set up correctly: $ sudo docker run hello-world
+- Install docker compose (click on the Linux tab) [Link to Docker Documentation](https://docs.docker.com/compose/install/)
+- Turn on docker `sudo docker-compose up -d`
+- Permission Denied Sad Path: `sudo usermod -aG docker $USER` (theoretically making it so you don’t have to use sudo, but let’s come back to this)
+Yarn bug: `wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
+https://github.com/MoveOnOrg/Spoke/blob/main/docker-compose.yml`
+- INITIAL YARN VERSION 1.22.17
+- `export NVM_DIR="$HOME/.nvm"`
+- command `-v nvm`
+- `source ~/.bashrc`
+- https://github.com/nvm-sh/nvm
+- `cp .env.example .env`
+- AFTER RUNNING: `npm install --build-from-resource`
+YARN VERSION IS: 
+- Run Yarn Dev, there’s enough defaults in the normal environment variables configurations file that you can just run it immediately.
+
+### Running on a different port.
+- Set up nginx
+- Go into nginx to configure the URL properly routes to the application when you go to it.
+- Link: All the steps for starting with a completely blank and empty server.
+- Go to section 4 to set up nginx
+
+
+## Set up the SSL
+- Most organizations have their own certificates.
+- But in case you don’t: Visit certbot.eff.org
+- Follow the instructions on the site
+- It assumes you’ve already set up your EngineX.
+- You’ll have to type in your contact information
+- Then it’s all up and running
+
+## Install Redis
+- Something set up by default within the Docker container.  It sets up a Redis instance.
+- We’re going to deploy redis separately within the server.
+- Since we’ve already broken the database option out of the Docker File.
+- But need to look for [Azure Cache for Redis instructions] (https://github.com/MoveOnOrg/Spoke/blob/main/docs/HOWTO_CONNECT_WITH_REDIS.md)
+
+## Install Postgres
+- In our documentation there are some information for setting this up.
```

Comment:
Nitpicky grammar correction: Should be "there is some information"

Reply:
Got it!